src/components/map/CreateGroupCard.tsx

```diff
@@ -11,39 +11,30 @@ import {
 } from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 import MapCard from '@nightlight/components/map/MapCard';
-import { CreateGroupCardProps, User } from '@nightlight/src/types';
+import { CreateGroupCardProps, Group, User } from '@nightlight/src/types';
 import { COLORS } from '@nightlight/src/global.styles';
 import CreateGroupCardStyles from '@nightlight/components/map/CreateGroupCard.styles';
 import CloseButton from '@nightlight/components/CloseButton';
-import { TEST_USERS } from '@nightlight/src/testData';
+import { useAuthContext } from '@nightlight/src/contexts/AuthContext';
+import { SERVER_URL } from '@env';
 
 const CreateGroupCard = ({ onClose }: CreateGroupCardProps) => {
-  const [availableUsers, setAvailableUsers] = useState<User[]>([]); // TODO: remove test data
+  const { userDocument, updateUserDocument } = useAuthContext();
+  const [availableUsers, setAvailableUsers] = useState<User[]>([]);
```

Comment:
```suggestion
  const [availableUsers, setAvailableUsers] = useState<User[]>([] as User[]);
```
Extra typing for the empty array :) Pretty sure it will default to any until assigned something

Reply:
not needed, since type is specified: `useState<User[]>`